manager.py

```diff
@@ -17,12 +17,19 @@ class TwitchManager:
     SECONDS_BETWEEN_UPDATE_STREAMS = 60
     SECONDS_BETWEEN_CREATE_BOTS = 15
 
-    def __init__(self, channels_amount, channels, log_filename=None):
+    def __init__(self, channels_amount, channels, log_filename=None, users_whitelist=None):
         self.bots = []
         self.channels_amount = channels_amount
         self.log_filename = log_filename
         self.channels = channels
-        self.db_logger = DatabaseLogger(settings.DATABASE['HOST'],
+        if settings.USERS is not None:
```

Comment:
Sorry I should have been more explicit here. You don't actually need the if here (neither the else and the contents inside it). In case `settings.USERS` is `None`, you will automatically be passing `None` as the last argument as well.